pkg/virt-dhcp/client.go

```diff
@@ -0,0 +1,46 @@
+package virtdhcp
+
+import (
+	"fmt"
+	"net/rpc"
+
+	"kubevirt.io/kubevirt/pkg/logging"
+)
+
+type VirtDHCPClient struct {
+	client *rpc.Client
+}
+
+func GetClient() (*VirtDHCPClient, error) {
+	conn, err := rpc.Dial("unix", DhcpSocket)
+	if err != nil {
+		logging.DefaultLogger().Error().Reason(err).Msgf("client failed to connect to virt-dhcp socket: %s", DhcpSocket)
```

Comment:
A patch has just been acked to change the logging part I think you will have to review that part to be coherent with what has been approved, no?